src/components/L2/Dropdowns/Dropdown.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Icon } from '@tlon/indigo-react';
+
+import './Dropdown.scss';
+
+export interface DropdownProps {
+  className: string;
+  children: any;
```

Comment:
Per [this reference](https://github.com/typescript-cheatsheets/react#useful-react-prop-type-examples) for React + Typescript, it's recommended to use `React.ReactNode` for `children` type declarations for improved type-checking